utils/appliance/__init__.py

```diff
@@ -625,11 +624,6 @@ def swap(self):
             value = None
         return value
 
-    @cached_property
```

Comment:
I'm planning to do that but a bit later . I even added callbacks support for that. 
However, the only thing that I'd like to get atm is fully functional register_event fixture.